docs/getting_started/installation.md

```diff
@@ -79,14 +79,27 @@ N rows in set (0.00 sec)
 ~~~
 
 !!! Note
-    iRODS is incompatible with versions 8.1, 8.2, and 8.3 of the MySQL ODBC Connector. You'll need to use v8.0 or v8.4.
+    While the MariaDB ODBC connector is more or less compatible with MySQL, iRODS does not currently support the MariaDB ODBC connector. Please use the MySQL ODBC connector instead. Additionally, iRODS is also incompatible with versions 8.1, 8.2, and 8.3 of the MySQL ODBC Connector. You'll need to use v8.0 or v8.4.
 
 !!! Note
     MySQL v8.0 and v8.4 use a default transaction isolation level of `REPEATABLE-READ`. You'll need to change this to `READ-COMMITTED` so that iRODS operates properly. This setting needs to be set globally. One way to achieve this is by starting MySQL with `--transaction-isolation=READ-COMMITTED`. For more information, see [https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html).
 
 !!! Note
     MySQL v8.0 and v8.4 queries operate in a case-insensitive manner by default. iRODS requires that data be handled in a case-sensitive way. One way to achieve this is by starting MySQL with a different character-set and collation scheme (e.g. `--character-set-server=utf8mb4` and `--collation-server=utf8mb4_0900_as_cs`). For more information, see [https://dev.mysql.com/doc/refman/8.0/en/charset.html](https://dev.mysql.com/doc/refman/8.0/en/charset.html).
 
+#### MariaDB:
+
+MariaDB can be used with the MySQL database plugin. Configuration is largely the same as MySQL (see the above section).
+
+!!! Note
+    iRODS does not currently support the MariaDB ODBC connector. Please use the MySQL ODBC connector instead. Additionally, iRODS is also incompatible with versions 8.1, 8.2, and 8.3 of the MySQL ODBC Connector. You'll need to use v8.0 or v8.4.
+
+!!! Note
+    MariaDB uses a default transaction isolation level of `REPEATABLE-READ`. You'll need to change this to `READ-COMMITTED` so that iRODS operates properly. This setting needs to be set globally. One way to achieve this is by starting MariaDB with `--transaction-isolation=READ-COMMITTED`. For more information, see [https://mariadb.com/kb/en/set-transaction/](https://mariadb.com/kb/en/set-transaction/).
+
+!!! Note
+    MariaDB queries operate in a case-insensitive manner by default. iRODS requires that data be handled in a case-sensitive way. One way to achieve this is by starting MariaDB with a different character-set and collation scheme (e.g. `--character-set-server=utf8mb4` and `--collation-server=utf8mb4_uca1400_as_cs`, or for versions of MariaDB prior to 10.10.1, `--character-set-server=utf8mb4` and `--collation-server=utf8mb4_bin`). For more information, see [https://mariadb.com/kb/en/character-sets/](https://mariadb.com/kb/en/character-sets/).
```

Comment:
These are the settings I used when running against the test suite. Notice the use of underscores just like with the transaction isolation level.
- `--character_set_server=utf8mb4`
- `--collation_server=utf8mb4_bin`

MariaDB didn't accept `utf8mb4_uca1400_as_cs` when I tried using that, but happily accepted the ones used above.

- https://mariadb.com/docs/server/ref/mdb/system-variables/character_set_server/
- https://mariadb.com/docs/server/ref/mdb/system-variables/collation_server/

Strangely enough, the docs I used were for MariaDB 10.6. They don't appear to have docs available for 11.4.

Reply:
Yes, MariaDB 10.6 doesn't have UCA-14.0.0 as it wasn't added until 10.10.1.
https://mariadb.com/kb/en/supported-character-sets-and-collations/

You can see in the testing environment that we are using `utf8mb4_bin` for 10.6 and `utf8mb4_uca1400_as_cs` for 10.11 and 11.4.

Also, as before, both dashes and underscores are valid:
https://mariadb.com/kb/en/server-system-variables/#character_set_server
https://mariadb.com/kb/en/server-system-variables/#collation_server

Reply:
In the testing environment, we are using `utf8mb4_uca1400_as_cs` for 10.11 and 11.4, and `utf8mb4_bin` for 10.6 (also we're using dashes in the option names) and it works just fine:
https://github.com/irods/irods_testing_environment/blob/main/projects/ubuntu-22.04/ubuntu-22.04-mariadb-10.6/docker-compose.yml#L11
https://github.com/irods/irods_testing_environment/blob/main/projects/ubuntu-22.04/ubuntu-22.04-mariadb-10.11/docker-compose.yml#L11
https://github.com/irods/irods_testing_environment/blob/main/projects/ubuntu-22.04/ubuntu-22.04-mariadb-11.4/docker-compose.yml#L11

Reply:
Let's stick with `utf8mb4_uca1400_as_cs` for now. If the tests fail during release testing of 4.3.3, we can discuss switching to `utf8mb4_bin`.

Reply:
Recent testing by @alanking has shown that the configuration captured by this documentation aligns with the testing environment.

Marking as resolved.